test/integration/pending.spec.js

```diff
@@ -119,7 +185,7 @@ describe('pending', function () {
 
     describe('in beforeEach', function () {
       it('should skip all suite specs', function (done) {
-        run('pending/skip-sync-beforeEach.fixture.js', args, function (err, res) {
```

Comment:
This test wasn't calling the `async` fixture, and it started failing when I fixed it. This is unrelated to my changes, as far as I can tell (see [this branch](https://github.com/bannmoore/mocha/tree/pending-async-beforeEach).

Reply:
But it's fixed now..?

Reply:
@boneskull I updated the test to pass _based on the current behavior_. If the expectations in that test are wrong, please let me know. :)